src/Datadog.Trace/TracerSubProcessManager.cs

```diff
@@ -12,6 +12,14 @@ internal class TracerSubProcessManager
     {
         private static Task _traceAgentMonitor;
         private static Task _dogStatsDMonitor;
+        private static Process _traceAgentProcess;
```

Comment:
Do we need to distinguish between these two processes? I'm thinking maybe we can have a list of processes and treat them all equally.

Reply:
We could do that, and split them out if we do need to treat them differently.